modules/social_features/social_group/social_group.module

```diff
@@ -838,12 +727,7 @@ function social_group_entity_view_alter(array &$build, EntityInterface $entity,
  * Implements hook_form_alter().
  */
 function social_group_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  $social_group_types = [
-    'open_group',
-    'closed_group',
-    'public_group',
-  ];
-
+  $social_group_types = [];
```

Comment:
@nechai Not sure why but locally this alter below doesn't work and it gives empty array.
As a result there are errors like:
```
TypeError: in_array(): Argument #2 ($haystack) must be of type array, null given in in_array() (line 877 of profiles/contrib/social/modules/social_features/social_group/social_group.module).
```
Since we use just flexible group in core now, can we just add  `flexible_group` by default without alter?

```
$social_group_types = [];
$social_group_types[] = 'flexible_group';
\Drupal::moduleHandler()->alter('social_group_types', $social_group_types);
```
cc @Kingdutch

Reply:
fixed